### PR TITLE
Update decompose to 2.2.x

### PR DESCRIPTION
**Pull request by alisherbu**

### CODE REVIEW

File: app-android/build.gradle.kts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@file:Suppress("DSL_SCOPE_VIOLATION")
```
Comment (arkivanov) [inline app-android/build.gradle.kts:1]: Why do we need this?

**Review by arkivanov — COMMENTED** [review]
Thanks for updating the template!

File: app-android/build.gradle.kts (hunk 1 above)
Comment (alisherbu) [reply app-android/build.gradle.kts:1]: I had an warning from AS. https://github.com/gradle/gradle/issues/18107
Is there another way without suppressing?

**Review by arkivanov — APPROVED** [review]

File: app-android/build.gradle.kts (hunk 1 above)
Comment (arkivanov) [reply app-android/build.gradle.kts:1]: I'm usually just ignoring it, but let's keep it.

**Pull request merged**